Library/Formula/libu2f-host.rb

```diff
@@ -0,0 +1,33 @@
+class Libu2fHost < Formula
+  desc "Libu2f-host implements the host-side of the U2F protocol."
```

Comment:
Please don’t start the description with the formula name. You could use something like `desc "Host-side of the U2F protocol"`.